pyproject.toml

```
@@ -236,6 +227,9 @@ filterwarnings = [
 # need to ensure build directories aren't excluded from recursion
 norecursedirs = []
 
+[tool.ruff.lint]
+select = ["I", "UP"]
```

```suggestion
select = ["E4", "E7", "E9", "F", "I", "UP"]
```
The select overrides the defaults, so you need to add them and the additional I and UP. :+1: